imports/api/flow-router-seo-config.js

```diff
@@ -18,4 +18,4 @@ SEO.setDefaults({
     'name="twitter:url"': 'http://808mix.com',
     'name="twitter:card"': 'The only station that loves you.'
   }
-});
+});*/
```

Comment:
Maybe we can keep (and not delete) this block and work toward updating the FlowRouterSEO plugin at a later time.